doc/reference/reference_lua/other.rst

```diff
@@ -140,59 +140,101 @@ and variables which are outside all modules.
 
 .. data:: package.path
 
-    This is a string that Tarantool uses to search for Lua modules,
-    especially important for ``require()``.
-    See :ref:`Modules, rocks and applications <app_server-modules>`.
+    Get directories in which Tarantool searches for Lua :ref:`modules <app_server-modules>`.
+    For example, these directories are used to find modules loaded using ``require()``.
+
+    Suppose the application has the following structure:
+
+    .. code-block:: none
+
+        /home/testuser/myapp
+        ├── .rocks/share/tarantool/
+        │   └── foo.lua
+        ├── init.lua
+        └── modules
+            └── bar.lua
+
+    If you :ref:`run the application <app_server-launching_app_binary>` using the ``tarantool`` command, ...
+
+    .. code-block:: console
+
+        tarantool init.lua
+
+    ... the ``/home/testuser/myapp`` path is added to search paths for modules automatically.
+    This means that to load the ``foo`` and ``modules.bar`` modules in ``init.lua``, you only need to add the corresponding ``require`` directives:
+
+    .. code-block:: lua
+
+        -- init.lua --
+        require('foo')
+        require('modules.bar')
+
+    To load modules placed in a non-default directory, use :ref:`setsearchroot() <other-package_setsearchroot>`.
+
 
 .. _other-package_cpath:
 
 .. data:: package.cpath
 
-    This is a string that Tarantool uses to search for C modules,
-    especially important for ``require()``.
-    See :ref:`Modules, rocks and applications <app_server-modules>`.
+    Get a path used by Tarantool to search for C :ref:`modules <app_server-modules>`.
+    For example, these directories are used to find modules loaded using ``require()``.
 
 .. _other-package_loaded:
 
 .. data:: package.loaded
 
-    This is a string that shows what Lua or C modules Tarantool
-    has loaded, so that their functions and members are available.
+    Show Lua or C modules loaded by Tarantool, so that their functions and members are available.
     Initially it has all the pre-loaded modules, which don't need
     ``require()``.
```

Comment:
Is it correct in the light of the https://github.com/tarantool/tarantool/commit/7e9051c44905dbe5c2396568ee252e09bf1963da change? Or I just moved the modules from the 'pre-loaded' to some other category?